packages/gamut-labs/src/landingPage/PageHero.tsx

```diff
@@ -14,49 +16,86 @@ const RightColumn = styled(Column)`
   }
 `;
 
+const VideoColumn = styled(Column)`
+  margin-left: 1rem;
+`;
+
 const Image = styled.img`
   width: 100%;
 `;
 
+const StyledVideo = styled(Video)`
+  align-self: center;
+`;
+
+export type ImageProps = {
+  src: string;
+  alt: string;
+};
+
+export type MediaProps =
+  | ({
+      type: 'image';
+    } & ImageProps)
+  | ({
+      type: 'video';
+    } & VideoProps);
+
 export type PageHeroProps = BaseProps & {
-  /**
-   * Hero image URL
-   */
-  imgSrc?: string;
-  /**
-   * Hero image alt text (for screen readers)
-   */
-  imgAlt?: string;
+  media?: MediaProps;
 };
 
 export const PageHero: React.FC<PageHeroProps> = ({
   title,
   desc,
   cta,
-  imgSrc,
-  imgAlt = '',
+  media,
   testId,
   onAnchorClick,
-}) => (
-  <LayoutGrid testId={testId}>
-    <Column
-      size={{
-        xs: 12,
-        sm: imgSrc ? 9 : 12,
-      }}
-    >
-      {title && <Title isPageHeading>{title}</Title>}
-      {desc && <Description text={desc} onAnchorClick={onAnchorClick} />}
-      {cta && (
-        <CTA href={cta.href} onCtaButtonClick={cta.onClick}>
-          {cta.text}
-        </CTA>
-      )}
-    </Column>
-    {imgSrc && (
-      <RightColumn size={3}>
-        <Image src={imgSrc} alt={imgAlt} />
-      </RightColumn>
-    )}
-  </LayoutGrid>
-);
+}) => {
+  const renderMedia = () => {
+    if (media && media.type === 'image') {
+      return (
+        <RightColumn size={3}>
+          <Image src={media.src} alt={media.alt} />
+        </RightColumn>
+      );
+    } else if (media && media.type === 'video') {
+      const videoArgs = omit({ ...media }, 'type');
```

Comment:
`omit` doesn't modify its parameter, so this should be fine:

```suggestion
      const videoArgs = omit(media, 'type');
```